packages/styles/scss/patterns/simplelongform/_simplelink.scss

```diff
@@ -0,0 +1,40 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@mixin simplelink {
+    .#{$prefix}--simplelink {
```

Comment:
I'm struggling to understand the need for this file.

Reply:
what do yo mean ?

Reply:
Why do we have this file for simple link when we a component `LinkWithIcon` that handles this for us?

Reply:
Ok, cool. I get that, so why aren't we using `LinkWithIcon` yet? Is there a reason we're waiting that I might have missed?